src/d3d9/d3d9_fixed_function.cpp

```diff
@@ -2241,8 +2236,9 @@ namespace dxvk {
     uint32_t atestSkipLabel = m_module.allocateId();
 
     // if (alpha_test) { ... }
+    uint32_t isNotAlways = m_module.opINotEqual(boolType, alphaFuncId, m_module.constu32(VK_COMPARE_OP_ALWAYS));
     m_module.opSelectionMerge(atestSkipLabel, spv::SelectionControlMaskNone);
-    m_module.opBranchConditional(alphaTestId, atestBeginLabel, atestSkipLabel);
+    m_module.opBranchConditional(isNotAlways, atestBeginLabel, atestSkipLabel);
```

Comment:
Why is that branch needed at all?

Reply:
It technically isn't, but we had it set up like this in the past to make some drivers happier.

We can get rid of it at some point once we guarantee folding everywhere.